db/migrate/20231207120538_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.bigint 'post_id'
+      t.text 'text'
```

Comment:
- [ ] Please kindly remove this line of code ` t.bigint 'post_id'` as suggested earlier. 
In Ruby on Rails, the t.bigint 'post_id' statement in a migration file alone doesn't explicitly indicate a foreign key relationship. This statement is used to create a column named post_id with a bigint data type, but it doesn't specify any constraints, such as foreign key constraints. You've actually one well by implementing it with `t.references :post, foreign_key: true`. 👍 

## Previous review's comment

<img width="552" alt="Untitled" src="https://github.com/Ayokunnumi1/blog_app/assets/103386183/0d4165db-b18f-4a8b-9cd6-3ccdd297450f">